File: src/main/webapp/js/service/PlayerService.js
app.service("playerService",function ($http) {
    this.findAll = function () {
        return $http.get("../getAll.do");
    }
    this.delete = function (ids) {
        return $http.post("../upDid.do",ids);
    }
    this.findSchool = function () {
        return $http.get("../getPlay.do");
    }
    this.findTeam = function () {
        return $http.get("../getTe.do");
    }
    this.save = function (entity) {
        return $http.post("../save.do",entity);
    }
})

File: src/main/java/com/lening/service/PlayerService.java
package com.lening.service;

import com.lening.entity.PlayerBean;
import com.lening.entity.SchoolBean;
import com.lening.entity.TeamBean;

import java.util.List;

public interface PlayerService {
    List<PlayerBean> getAll();

    void save(PlayerBean playerBean);

    List<SchoolBean> getSch();

    void upDid(Long[] ids);

    List<TeamBean> getTe();
}
